dell/precision/3490/intel/default.nix

```diff
@@ -0,0 +1,15 @@
+{
+  imports = [
+    ../../../../common/cpu/intel/meteor-lake
+    ../../../../common/pc/laptop
+  ];
+
+  boot.initrd.availableKernelModules = [
+    "nvme"
+    "sd_mod"
+    "thunderbolt"
+    "usb_storage"
+    "vmd"
+    "xhci_pci"
+  ];
```

Comment:
Please only include modules not detected by nixos-generate-config

```suggestion
```

Some of those are also included in NixOS by default (when nixos-generate-config includes the not-detected profile).